using-user-input.py

```diff
@@ -0,0 +1,11 @@
+# A program to print 'Hello World' by force typing it by the user himself
+
+def hello_world():
+    str = input("Enter 'Hello World' exactly: ")
+    if str == 'Hello World':
```

Comment:
It's supposed to print `Hello, World!` Kindly update the comparison to match it.